p2p/protocol/identify/pb/identify.proto

```diff
@@ -37,6 +39,11 @@ message Identify {
   // a delta update is incompatible with everything else. If this field is included, none of the others can appear.
   optional Delta delta = 7;
 
+  // more is set to true by a peer to indicate to the receiver that the signed record is absent in the current message
+  // and the peer should read the next message from the stream to fetch it.
+  // This is done because messages with signed records can exceed the legacy message size of 2K bytes.
+  optional bool more = 9 [(gogoproto.nullable) = false];
```

Comment:
What's this nullable thing? I would advise against using gogo specific stuff in the protobuf decls, as there are other implementations in different languages that want to use the same protobuf file.

Reply:
This is just an annotation to tell gogo how to generate the go code for this protobuf. Other protobuf implementations will ignore annotations they don't understand.